web-app/HeimScripts/heatmap/run.R

```diff
@@ -104,6 +106,30 @@ buildFields <- function(df){
   return(df)
 }
 
+writeDataForZip <- function(df, patientIDs) {
+  pidCols    <- as.character(patientIDs)
+  t          <- df
+  t[pidCols] <- lapply(t[pidCols], function(v) { (v - df$MEAN) / df$SD })
```

Comment:
Okay, if Piotrs PR (#71) is merged, then there is even another place where zscores is calculated, and then the table is not pivoted so it should be usable without major operations on it